set_version.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+import json
+import os
+
+def main():
+    with open('package.json', 'r+') as f:
```

Comment:
You should open this writable, should you not?

Reply:
That's what 'r+' means I'm told

Reply:
Yes, sorry. You need read/write, makes sense.